code/WorkInProgress/rewardsLocker.dm

```diff
@@ -1179,3 +1179,63 @@
 					src.player.claimed_rewards.Add(S.type)
 			else
 				boutput(usr, "<span class='alert'>Redemption of \"[S.title]\" failed.</span>")
+////////navygreyhop////////
+/datum/achievementReward/ntso_commander
+	title = "(Skin set) Bland Bureaucrat"
+	desc = "Will change the skin of suits, naval jackets, and Head of personal uniforms."
+	required_medal = "Unperson"
+	once_per_round = 0
+
+	rewardActivate(var/mob/activator)
+		if (ishuman(activator))
```

Comment:
You can probably make the function clearer by making this

```
if (!ishuman(activator))
   boutput("whatever message")
   return FALSE

// Whatever other code you want to run in the other case that they are human
```

Reply:
I just coped the cap code tbh, it could be more clear but i dont want to mess it up